lib/alephant/logger/level.rb

```diff
@@ -0,0 +1,35 @@
+module Alephant
+  module Logger
+    LEVELS = %i(debug info warn error).freeze
+
+    class Level
+      def initialize(message_level)
+        @message_level = message_level
+      end
+
+      def logs?(desired_write_level)
+        message_level_index >= desired_write_level_index(desired_write_level)
+      end
+
+      private
+
+      def desired_write_level_index(desired_write_level)
```

Comment:
Why don't we support strings?

Reply:
No specific reason, we could add string support.